Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
This should be in speed optimised. PWM output is part of the GYROPID loop. When running at 32kHz every machine cycle counts - there is less than 31 microseconds to complete the loop.